components/inputs/input-checkbox.js

```diff
@@ -114,6 +115,7 @@ class InputCheckbox extends InputInlineHelpMixin(FocusMixin(SkeletonMixin(LitEle
 			 * @type {string}
 			 */
 			value: { type: String },
+			_hasSupporting: { type: Boolean, attribute: '_has-supporting', reflect: true },
```

Comment:
If we don't want to sprout a "private" property/attribute on the host, we could conditionally apply a different CSS class to that supporting container `div`.

Reply:
Yeah I like that approach better -- thanks.

Reply:
Ok done!